### PR TITLE
Add Gridlines and size display

### LINKED ISSUE #87: [5] Change to QOpenGLWidget
We have bugs with the current OpenGLWindow implementation, switch to a widget.

### PR DESCRIPTION
**Pull request by tigerman9854**
- Switched from QWindow to QOpenGLWidget
- Fixed MANY issues that came with this change^
- Added axes and text on the screen
- Reworked some controls to feel better

Finishes half of #41 

Closes #87

### CODE REVIEW

**Comment by tigerman9854** [comment]
image

**Comment by tigerman9854** [comment]
image

File: ModelViewer/ModelViewer/Axes.h [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+     0.5f, -0.5f
+};
+
+static const GLfloat colors[] = {
+    1.0f, 0.0f, 0.0f,
+    0.0f, 1.0f, 0.0f,
+    0.0f, 0.0f, 1.0f
+};
+
+static const GLfloat grid[] = {
```
Comment (richjoregonstate) [inline ModelViewer/ModelViewer/Axes.h:32]: You think it's worthwhile to allow the user to have a grid of a set size? Could be another issue. 

At the very least I'm going to add a 'toggle grid' option to #80 :)

**Review by richjoregonstate — APPROVED** [review]
Looks great!

File: ModelViewer/ModelViewer/Axes.h (hunk 1 above)
Comment (tigerman9854) [reply ModelViewer/ModelViewer/Axes.h:32]: Oh yeah, this was a very primitive implementation of the grid.  I'm planning to add auto-resizing of the grid based on zoom level.

**Pull request merged**